### DISCUSSION TITLE
How to install older version of Storybook, lets say v3

### DISCUSSION THREAD

**Discussion by anatoly314** [storybookjs/storybook#16741] [Help] [answered]
Hi,
We have a legacy code that we can't upgrade right now, but I'd like to add the Storybook to it. The problem is that the project was built with WebPack v2, node v12, etc... So, when I try to install the most recent storybook it doesn't work, complaining that it lacks a lot of dependencies. 
Can I still install Storybook v3? I think it will be a more flawless process. 
Thank you.

**Comment by shilman** [comment, marked as answer]
Your best bet is to try to follow the manual instructions described in the 3.x docs which are no longer published but are available here and adjusting the NPM package versions accordingly:

https://github.com/storybookjs/storybook/blob/release/3.4/docs/src/pages/basics/guide-react/index.md

**Reply by anatoly314** [reply]
Thank you very much. Some tweaking is still required but it works like a charm. I've wanted to add Storybook to the project for half a year already but didn't have time to dig into it till today. It will reduce dramatically time to develop new features.

**Reply by shilman** [reply]
Wonderful news! That said I hope you can upgrade eventually. We're working hard to make SB better in every release, and when you use v3 you're missing out on years of improvements!

**Reply by anatoly314** [reply]
I'm in the middle of implementing this part of the Pragmatic Programmer book :)
https://flylib.com/books/en/1.315.1.16/1/
